setup.cfg

```diff
@@ -0,0 +1,58 @@
+# SPDX-FileCopyrightText: 2019 The curiu authors
+#
+# SPDX-License-Identifier: LicenseRef-ISC-curiu
+
+[metadata]
+name = cuiru
+license = ISC
+license_file = LICENSE.txt
+description = Experimental code for the conda package mananger.
+long_description = file: README.md, AUTHORS.md, CHANGELOG.md
+long_description_content_type = text/markdown; charset=UTF-8; variant=GFM
+url = https://github.com/curiu/curiu
+author = Marcel Bargull
+author_email = marcel.bargull@udo.edu
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Intended Audience :: Developers
+    License :: OSI Approved :: ISC License (ISCL)
+    Natural Language :: English
+    Operating System :: MacOS
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: System :: Software Distribution
+
+[options]
+install_requires =
+    conda ==4.7.12
+python_requires = >=3.7
+setup_requires =
+    setuptools >=38.6.0  # support for long_description_content_type
+    # cffi >=1.0.0
+# include_package_data = True
+packages = find:
+package_dir =
+    = src
+
+[options.packages.find]
+where = src
+
+[options.entry_points]
+console_scripts =
+    curiu = curiu.ui.cli:main
+
+[versioneer]
+VCS = git
```

Comment:
for versioning we might use ``setuptools-scm``. IMHO, it is a great tool 
https://github.com/pypa/setuptools_scm/

Reply:
It's been a while that I tried `setuptools_scm` (maybe late 2017/early 2018?). There were some things that bothered me, but I can't quite remember...
One thing I do remember is that you have to use `pkg_resources` (i.e., part of `setuptools`) to retrieve the version at runtime. I guess nowadays you can use `importlib_metadata` (or the CPython 3.8 provisional `importlib.metadata`) for this now..
With `versioneer` you just don't have any external dependency (well, apart from Python's builtin `json` module) in your releases. (Downside is the need to vendor `versioneer` in the repo..)

Reply:
Sure, we can go for it them!
Both are fine  :)